libqtile/widget/base.py

```diff
@@ -203,6 +202,18 @@ def _configure(self, qtile, bar):
         if not self.configured:
             self.configured = True
             self.qtile.call_soon(self.timer_setup)
+            self.qtile.call_soon(self._eventloop_started)
+
+    def _eventloop_started(self):
```

Comment:
I guess we probably want to do async stuff most of the time with this method, so why not making it async in the first place? That would make the diff simpler on the widgets you used it for. :)

Reply:
D'oh! Yes, much better idea. 

My expectation was that this method would just be for asyncio stuff so this makes much more sense than the approach I adopted.

Reply:
Might change the name back to `_config_async` as that would make more sense with this change.

Reply:
I now remember why I didn't do it this way: you can't do `asyncio.create_task` in `_config` as the eventloop may not have started so you get an error. However, I do think having an async method makes sense so I'll fix that.